_episodes/04-drawing-bitwise.md

```diff
@@ -162,17 +151,20 @@ Here is what our constructed mask looks like:
 > 
 > ![Sample shapes](../fig/03-draw-practice.jpg)
 > 
-> Circles can be drawn with the `cv2.circle()` function, which takes five 
-> parameters: the image to draw on, the (x, y) point of the center of the 
-> circle, the radius of the circle, the color for the circle, and the thickness
-> of the line used, or -1 to draw a filled circle. 
+> Circles can be drawn with the `skimage.draw.circle()` function, which takes three
+> parameters: x, y point of the center of the circle, and the radius of the
+> filled circle. There is an optional `shape` parameter that can be supplied to
+> this function. It will limit the output coordinates for cases where the circle
+> dimensions exceed the ones of the image.
 > 
-> Lines can be drawn with the `cv2.line()` function, which takes four parameters:
-> the image to draw on, the (x, y) coordinate of one end of the segment, the 
-> (x, y) coordinate of the other end of the segment, and the color for the line.
+> Lines can be drawn with the `skimage.draw.line()` function, which takes four
+> parameters: the image to draw on, the (x, y) coordinate of one end of the
+> segment, the (x, y) coordinate of the other end of the segment, and the color
+> for the line. There is also an anti-aliased line-drawing function that avoids
+> step-artifacts: `skimage.draw.line_aa()`.
```

Comment:
This last sentence seems jargony to me. Will "anti-aliasing" and "step-artifacts" be terms learners are familiar with? If not, we should define them here and in the glossary.

Reply:
Alternatively we could just delete the mention of this sentence. Although knowing about alias artifacts doesn't hurt, it might be better introduced somewhere else (or maybe in a bit of detail here). What would you prefer @ErinBecker?